data/json/items/ammo/308.json

```diff
@@ -17,7 +17,7 @@
     "ammo_type": "308",
     "casing": "308_casing",
     "range": 65,
-    "damage": { "damage_type": "bullet", "amount": 60, "armor_penetration": 2 },
+    "damage": { "damage_type": "bullet", "amount": 54, "armor_penetration": 2 },
```

Comment:
I don't see corresponding changes to the 308 guns in this PR. They were in the (now closed) rebalance PR. Not sure if it's intentional that the edits to the weapons aren't here?

Reply:
> I don't see corresponding changes to the 308 guns in this PR. They were in the (now closed) rebalance PR. Not sure if it's intentional that the edits to the weapons aren't here?

You know, I suspected I forgot to port something over when creating this PR. Thanks for pointing it out. On the subject of the two 12.3mm LN rifles, my singular question is what should their barrel lengths be? I suppose I can look at their respective lengths and try to make a guestimation based on that.